ExerciseFinalREADTHIS.R

```diff
@@ -0,0 +1,35 @@
+rm(list=ls())
+
+#Sort unique combinations of gender and years experience by first gender and then years experience
+wages=read.csv(file="wages.csv",header=TRUE,stringsAsFactors=FALSE)
+wages2=unique(wages[,1:2])

```

Comment:
Be careful naming things like this. If you come back later you may not know what you were trying to do.